2020/day3.clj

```diff
@@ -1,89 +0,0 @@
-(ns advent-of-code.2020.day3
-  (:require [clojure.string :as string]))
-
-(defn file->seq
-  [filename]
-  (->> (slurp filename)
-       (string/split-lines)))
-
-(defn next-location
-  [loc [right down]]
-  [(+ (first loc) right)
-   (+ (second loc) down)])
-
-(defn locations-s
```

Comment:
This is complecting slope (which does make sense for determining your path/locations) with grid height, which is NOT a necessary part of determining your path/locations).

i.e., these are currently not generic locations for a slope... these are locations for a specific grid

Reply:
> This is complecting slope (which does make sense for determining your path/locations) with grid height, which is NOT a necessary part of determining your path/locations).
> 
> i.e., these are currently not generic locations for a slope... these are locations for a specific grid

rewrote in second PR.